### PR TITLE
Strengthen trading model

### PR DESCRIPTION
**Pull request by jkrvivian**

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -299,17 +299,16 @@ \subsection{TA - endpoint}
...
-Trading Models of Decentralized Data Marketplace
-\subsubsection{Set Up}
-All participants need to register on TangleID\cite{TangleID}, a self-sovereign identity system built on top of MAM, enables authentication without any third party via a decentralized identifiers and a public/private key pair for authentication.
+\section{Decentralized Subscription-based Data Trading Case Study}
+In this section, we illustrate how participants can join the data subscription platform, start subscribing, cancel subscription, and request refund.
 
-\subsubsection{Launch Data Products}
-To launch a data product, data providers need to create a MAM channel/endpoint and a Product Contract. However, considering IoT devices have no ability to handle both uploading data to MAM and interacting Ethereum smart contracts while collecting continuous data with low computing power, these works are offloaded to brokers. 
+\subsection{Prerequisite}
+All participants are required to register on TangleID to get DID document and public/private key pair for sensitive contents exchange and authentication. An Ethereum account is also needed to interact with smart contracts and transfer Ethereum tokens.
```
Comment (jserv) [inline paper.tex:306]: 需要說明 TangleID 的註冊和典型中心化架構的行為落差

Comment (jkrvivian) [reply paper.tex:306]: 或是在第三章的 TangleID 說明？

Comment (jserv) [reply paper.tex:306]: Yes, please go ahead.

**Comment by jkrvivian** [comment]
@jserv , I just thought about the unsubscription process these days, and I mentioned the cancellation fee, yet I still need to study more about this to check if it's reasonable or if there's other things that need to be awared of. 

Let's talk about it on Monday! :rocket:

**Pull request merged**